setup.py

```diff
@@ -25,48 +25,48 @@ def read(*filenames, **kwargs):
 
 root = os.path.dirname(os.path.realpath(__file__))
 version_module = imp.load_source(
-    'version', os.path.join(root, 'nengo_spa', 'version.py'))
-testing = 'test' in sys.argv or 'pytest' in sys.argv
+    "version", os.path.join(root, "nengo_spa", "version.py"))
```

Comment:
Is this supposed to be changed from `nengo_spa` to `nengo-spa`?

Reply:
No, this refers to the `nengo_spa` directory relative to the root ([this one](https://github.com/nengo/nengo-spa/tree/master/nengo_spa))